internal/api/api.go

```diff
@@ -10,19 +10,35 @@ import (
 	"net/http"
 )
 
-type IdlerAPI struct {
+type IdlerAPI interface {
+	// TODO Document the API
```

Comment:
@vpavlin maybe you can jump in here and help me with the docs. In particular _Builds_ seems to be a bit odd. At least in terms of naming. 

Do we have to expose this?

Reply:
The Builds should be probably renamed to Users and no, we don't have to expose it - it was purely for debugging purposes, so we could make clear in comments that people should not use it and remove it when we verify things work.

Reply:
> The Builds should be probably renamed to Users 

+1

> and no, we don't have to expose it - it was purely for debugging purposes, 

that's what I thought. If we had to shuttle around so complex data structures, we would have to look indeed into a way to generate a REST API client. I guess that would be GOA then, but I don't think we need to go down this road as yet. 

> so we could make clear in comments that people should not use it 

+1

> and remove it when we verify things work.

+1